README.md

```diff
@@ -50,4 +50,4 @@ Export to:
 
 If you get an error message about missing shapely, just do
 
-`sudo apt-get install python-shapely`.
\ No newline at end of file
+`sudo apt-get install python-shapely`.
```

Comment:
no idea what github did here, i did not mean to touch that line. seems unchanged though?